pkg/kf/commands/config/config.go

```diff
@@ -277,8 +275,15 @@ func getRestConfig(p *KfParams) *rest.Config {
 	return restCfg
 }
 
-func initKubeConfig(p *KfParams) {
-	if p.KubeCfgFile == "" {
-		p.KubeCfgFile = filepath.Join(homedir.HomeDir(), ".kube", "config")
+func initKubeConfig() KfParams {
+	p := KfParams{}
+
+	p.KubeCfgFile = filepath.Join(homedir.HomeDir(), ".kube", "config")
+
+	// Override path to Kube config if env var is set
+	if configPathEnv, ok := os.LookupEnv("KUBECONFIG"); ok {
+		p.KubeCfgFile = configPathEnv
```

Comment:
Does this work if KUBECONFIG has multiple paths (separated with a ':'): https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/#the-kubeconfig-environment-variable

Reply:
I guess no. That's what `clientcmd.NewDefaultClientConfigLoadingRules()` does: Split envVar files if KUBECONFIG has multiple paths

Reply:
Now, it works for multi paths: https://github.com/google/kf/pull/525/files#diff-bb99e7b0d56a5e572b70ad586e5bf07dR279-R284